CMakeLists.txt

```diff
@@ -17,6 +17,7 @@ option(MATERIALX_BUILD_DOCS "Create HTML documentation using Doxygen. Requires t
 option(MATERIALX_BUILD_GEN_GLSL "Build the GLSL shader generator back-end." ON)
 option(MATERIALX_BUILD_GEN_OSL "Build the OSL shader generator back-end." ON)
 option(MATERIALX_BUILD_GEN_MDL "Build the MDL shader generator back-end." ON)
+option(MATERIALX_BUILD_GEN_ESSL "Build the ESSL shader generator back-end." ON)
```

Comment:
For generation itself no. For compilation testing yes but as discussed I think adding something like `glslValidator` to do this can come as a second step.